src/SectionsNavigation.Uno.WinUI/SectionsNavigation.Uno.WinUI.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0-windows10.0.18362;netstandard2.0;net6.0-android;net6.0-ios;</TargetFrameworks>
+    <LangVersion>10.0</LangVersion>
+    <!-- Ensures the .xr.xml files are generated in a proper layout folder -->
+    <GenerateLibraryLayout>true</GenerateLibraryLayout>
+    <RootNamespace>Chinook.SectionsNavigation</RootNamespace>
+    <Authors>nventive</Authors>
+    <Company>nventive</Company>
+    <AssemblyName>Chinook.SectionsNavigation.Uno.WinUI</AssemblyName>
+    <PackageId>Chinook.SectionsNavigation.Uno.WinUI</PackageId>
+    <Description>Unified cross-platform tools to perform ViewModel-based navigation.</Description>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <PackageTags>navigation;mvvm;maui;winui;ios;android;chinook</PackageTags>
```

Comment:
make sure you add maui and winui as tags in the other csproj too (like Abstractions and non-Uno csprojs).